keanu-project/src/test/java/io/improbable/keanu/vertices/dbltensor/nonprobabilistic/binary/BinaryOperationHelper.java

```diff
@@ -0,0 +1,94 @@
+package io.improbable.keanu.vertices.dbltensor.nonprobabilistic.binary;
+
+import io.improbable.keanu.vertices.dbltensor.DoubleTensor;
+import io.improbable.keanu.vertices.dbltensor.DoubleTensorVertex;
+import io.improbable.keanu.vertices.dbltensor.KeanuRandom;
+import io.improbable.keanu.vertices.dbltensor.Nd4jDoubleTensor;
+import io.improbable.keanu.vertices.dbltensor.nonprobabilistic.ConstantTensorVertex;
+import io.improbable.keanu.vertices.dbltensor.nonprobabilistic.diff.TensorDualNumber;
+import io.improbable.keanu.vertices.dbltensor.probabilistic.TensorUniformVertex;
+
+import java.util.function.BiFunction;
+
+import static org.junit.Assert.assertEquals;
+
+public class BinaryOperationHelper {
```

Comment:
We haven't called anything else `Helper` yet, why don't we call this something like `TensorBinaryOperators`?